plugin/cache/README.md

```diff
@@ -24,8 +24,8 @@ cache [TTL] [ZONES...]
 * **ZONES** zones it should cache for. If empty, the zones from the configuration block are used.
 
 Each element in the cache is cached according to its TTL (with **TTL** as the max).
-A cache is divided into 256 shards, each holding up to 512 items by default - for a total size
-of 256 * 512 = 131,072 items.
+A cache is divided into 256 shards, each holding up to 19 items by default - for a total size
+of 256 * 19 = 9728 items.
```

Comment:
it is 39 no ?
(256 * 19 = 4864)

Reply:
maybe good to say it is the closest value to 10000 items .. (just because choosing 39 or 9728 seems really weird)